modules/client-configuration/pages/clients-opensuse.adoc

```diff
@@ -50,12 +47,6 @@ ifeval::[{uyuni-content} == true]
 | Updates Channel
 | Other Channels
 
-| {opensuse} Leap 15.1
-| opensuse_leap15_1
-| opensuse_leap15_1-uyuni-client
-| opensuse_leap15_1-updates
-| opensuse_leap15_1-non-oss and opensuse_leap15_1-non-oss-updates
-
```

Comment:
If we are going to treat 15.1 and 15.2 as different versions, then it's OK to remove 15.1 but then we need to add 15.3.

I would just say "Leap 15" and that's it. In the end, even the client tools are for Leap 15 GA and they will work on Leap 15, Leap 15.1, Leap 15.2 and Leap 15.3.

Reply:
No, you can't say `leap15` for the channel labels. The channel labels are different. Not just for the client tools (even if the URL for the repo is the same), but most importantly for the channels that come from openSUSE.

Another story is if we say `leap15_X` and tell users to replace "X" with the correct revision. That would work.

Reply:
I think for the channel labels we need to be specific, that's why we created the tables. For the support matrices, etc, being more vague is OK.

Reply:
OK